code/src/components/ThoughtsList.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable no-underscore-dangle */
+import React from 'react';
+import { formatDistance } from 'date-fns';
+
+// eslint-disable-next-line no-unused-vars
+const ThoughtsList = ({ loading, thoughtsList, setThoughtsList, onLikesIncrease }) => {
+  if (loading) {
+    return <h1>Loading in progress...</h1>
+  }
+
+  return (
+    <section className="thoughtWrapper">
+      <div>
+        {thoughtsList.reverse().map((thought) => (
```

Comment:
In the happy thought API (compared to the test API) the thoughts already comes in the right order. Therefore you don't need to reverse it.